packages/components/TabList/src/Tab/Tab.types.ts

```diff
@@ -109,6 +109,14 @@ export interface TabProps extends Omit<PressablePropsExtended, 'onPress'> {
    */
   icon?: IconProps;
 
+  /**
+   * Conditional flag to disable the default Tab's onLayout handler from sending animation values to the TabList.
+   *
+   * If you want to set the onLayout handler from useTabAnimation() on your own custom element, this must be set to
+   * true so that animations run correctly.
+   */
+  calculateAnimationValuesExternally?: boolean;
```

Comment:
Why do we need a separate bool for this? Is it not possible to check if onLayout is defined and alter behavior accordingly? This feels like it's leaking implementation details.